### DISCUSSION TITLE
CSRF verification failed when login AWX

### DISCUSSION THREAD

**Discussion by azhinu** [gravitational/teleport#8569] [Q&A]
Hello everyone. I tried to add AWX as application to teleport, but when I try to log in, I'm getting `CSRF verification failed` error. If I'll log in same domain without teleport (on another port), I can continue login session through teleport. 
Both Teleport and AWX are working in K8s.

I tired to log in with Min browser, Chrome and Safari, but no success.

**Comment by webvictim** [comment]
Does AWX have a public address setting that you can configure - like a concept of its own hostname? If so, try setting it to `awx.teleport.example.com` or whatever the URI is that you're using to access it via Teleport application access so that it sets the cookies correctly.

**Reply by azhinu** [reply]
AWX doesn't need to specify hostname in service settings. I can access the same AWX instance with any address.

**Reply by webvictim** [reply]
Right, but its CSRF protection settings are preventing it from working correctly, which is why I was asking whether it's possible to set it.

**Reply by azhinu** [reply]
It correctly works with any hostname without any settings tune. As I wrote above, with `awx.tele.example.com` through teleport I can't log in, with `awx.tele.example.com:444` without teleport I can log in and with `awx.tele.example.com` but without teleport I can log in too.

**Reply by matjazp** [reply]
Any news about this? I too stumbled on web app that has it's own login page and fails if used with Teleport. I can imagine that there are a lot of these legacy apps around. What to to in cases like this?
